demos/src/accordion.mustache

```diff
@@ -0,0 +1,38 @@
+<div data-o-component="o-expander" class="o-expander items" data-o-expander-shrink-to="3" data-o-expander-count-selector="li">
+	<h2>Simple List</h2>
+	<ul class="o-expander__content">
+		<li>item</li>
+		<li>item</li>
+		<li>item</li>
+		<li>item</li>
+	</ul>
+	<a href='#' class="o-expander__toggle o--if-js"></a>
+</div>
+
+<div data-o-component="o-expander" class="o-expander items" data-o-expander-shrink-to="2" data-o-expander-count-selector="li" data-o-expander-accordion="true">
+	<h2>Accordion behaviour</h2>
+	<ul class="o-expander__content">
```

Comment:
The behaviour is independent of where the elements are, as it depends of the `data-expand` attributes, so I can move them out of the `<ul>`